tests/std/tests/P0768R1_spaceship_operator/test.cpp

```diff
@@ -15,15 +15,11 @@ constexpr bool test_ord(T val) {
     assert((0 == val) == (Z == comp::equal));
     assert((val != 0) == (Z != comp::equal));
     assert((0 != val) == (Z != comp::equal));
-#ifdef __cpp_impl_three_way_comparison
     assert(((val <=> 0) == 0) == (Z == comp::equal));
     assert(((0 <=> val) == 0) == (Z == comp::equal));

```

Comment:
Also test `!= 0`?

Reply:
Ditto "these tests and the proposed additions are redundant combinations of expressions tested elsewhere."